pypeit/spectrographs/spectrograph.py

```diff
@@ -473,7 +473,7 @@ def get_rawimage(self, raw_file, det):
                 oscansec_img = pix_img.copy()
 
         # Return
-        return raw_img, hdu, exptime, rawdatasec_img, oscansec_img
+        return raw_img, headarr, exptime, rawdatasec_img, oscansec_img
```

Comment:
Now it is a headarr. Shouldn't this be the same for all instruments that have multiple extensions?